pytoda/smiles/polymer_language.py

```diff
@@ -84,6 +84,21 @@ def update_entity(self, entity: str) -> None:
         ), f'Unknown entity was given ({entity})'
         self.current_entity = entity.capitalize()
 
+    def add_start_stop_tokens(self, tokenized_smiles: Iterable[str]
+                              ) -> Iterable[str]:
+        """Add the start and stop tokens to the smiles
+
+        Args:
+            smiles (Iterable[str]): tokenized smiles
+
+        Returns:
+            Iterable[str]: tokenized smiles with the extra start and stop
+        """
+        return [
+            '<' + self.current_entity.upper() + '_START>', *tokenized_smiles,
```

Comment:
It's better to use the `start_entity_tokens` and  `stop_entity_tokens` attributes here.

Reply:
Could you expand on this? Would you pass the entity as an argument to the function?

Reply:
can't you do sth like:
```
return [
            self.start_entity_tokens[idx], *tokenized_smiles,
            self.stop_entity_tokens[idx]
        ]
```
where you retrieve `idx` by checking the current entity?

Reply:
Hmm... okay, pushed in 28303abe6e17a01d2a8c95075982aeaf5c2216ad

Reply:
Great, think that's the best solution also for the skipping of the tokens.